pubsub/system-test/iam.test.js

```diff
@@ -46,7 +47,7 @@ describe('pubsub:iam', function () {
       program.getTopicPolicy(topicName, function (err, policy) {
         assert.ifError(err);
         assert(policy);
-        assert(console.log.calledWith('Got topic policy:', policy));
+        assert(console.log.calledWith('Got policy for topic: %s', topicName));
```

Comment:
IIRC you said you'd clean up the asserts later. (They definitely need to be brought up to our new standard.)